bokehjs/src/lib/core/ui_events.ts

```diff
@@ -105,7 +105,10 @@ export class UIEvents implements EventListenerObject {
   readonly keydown:      UISignal<KeyEvent>     = new Signal(this, 'keydown')
   readonly keyup:        UISignal<KeyEvent>     = new Signal(this, 'keyup')
 
-  private readonly hammer = new Hammer(this.hit_area, {touchAction: 'auto'})
+  private readonly hammer = new Hammer(this.hit_area, {
+    touchAction: 'auto',
+    inputClass: Hammer.TouchMouseInput // https://github.com/bokeh/bokeh/issues/9187
```

Comment:
Tests will fail without the trailing comma.

Reply:
Oh, sure. It's weird that linter hasn't notified me about it. I see a rule in there.